yarn-project/package.json

```diff
@@ -8,6 +8,7 @@
     "docs": "typedoc --out docs/dist && cd docs && yarn serve",
     "formatting": "yarn workspaces foreach -v run formatting",
     "formatting:fix": "yarn workspaces foreach -v run formatting:fix",
+    "format": "yarn prettier --cache -w .",
```

Comment:
hmm, seems weird next to formatting. Maybe 'format-staged'?

Reply:
Hmm it should be format-changed I guess. Merging for now, but feel free to rename.